examples/convert_group_studies.py

```diff
@@ -26,53 +22,42 @@
 
 import mne
 from mne_bids import write_raw_bids, make_bids_basename
-from mne_bids.datasets import fetch_faces_data
 from mne_bids.utils import print_dir_tree
 
 ###############################################################################
 # And fetch the data.
-# .. warning:: This will download 7.9 GB of data for one subject!
-
-subject_ids = [1]
-runs = range(1, 7)
```

Comment:
can we not completely scrap this? The purpose was to show people how they could bootstrap what exists for multiple runs and subjects. I don't think it's obvious to everyone how it can be done. The idea is to go from one organization to another more standardized one.

Reply:
I reset the ds000117 example, it's like it was before. See https://github.com/mne-tools/mne-bids/pull/254/commits/78a216d2e864af3b8d1b81d44105c901809ed3c1